test/main.js

```diff
@@ -56,4 +56,7 @@ before('initialization', function () {
 	LoginServer.disabled = true;
 
 	Ladders.disabled = true;
+
+	// eslint-disable-next-line no-unused-vars
+	const toID = require('../.sim-dist/dex').toID;
```

Comment:
?

Reply:
If you're trying to create a global, this is definitely the wrong way.

How was the global created in the past? Did including `sim/` just do it by default?

Reply:
I couldn't find anything about how it was created. I pretty much pushed this to be like "this was my best guess, anyone know better?", honestly.

Reply:
In theory, if you just remove this, nothing else will change. Right? Just remove these three lines, you don't need to do anything else.

Reply:
Seems fine. My bad!